tools/install.sh

```diff
@@ -32,9 +32,9 @@ sed -i -e "/export PATH=/ c\\
 export PATH=\"$PATH\"
 " ~/.zshrc
 
-if [ "$SHELL" != "$(which zsh)" ]; then
+if [ "$SHELL" != "$(grep /zsh$ /etc/shells | tail -1)" ]; then
```

Comment:
Can you change this to 
```sh
if [ "${SHELL:t}" != "zsh" ]; then
```
as it is now, if `$SHELL` is zsh but not the last one in `/etc/shells` it will trigger a `chsh`.